tests/EventListener/StatsSubscriberTest.php

```diff
@@ -15,13 +15,16 @@
  */
 final class StatsSubscriberTest extends WebTestCase
 {
+    /** @var PersisterInterface&\PHPUnit\Framework\MockObject\MockObject */
     protected $persister;
 
+    /** @var \PHPUnit\Framework\MockObject\MockObject&ControllerEvent */
     protected $controllerEvent;
 
+    /** @var \PHPUnit\Framework\MockObject\MockObject&Request */
```

Comment:
As I said above, if it's supported it would be better to use this form:
```suggestion
    /** @var \PHPUnit\Framework\MockObject\MockObject<Request> */
```

Reply:
See my comment above